gym_minigrid/envs/negated_goals.py

```diff
@@ -32,13 +32,14 @@ def __init__(
         }
 
         self.base_templates = [
-            "Go to the object that is <not><desc>",
-            "Navigate to the object that is <not><desc>",
-            "Find the object that is <not><desc>",
+            # "Go to the object that is <not><desc>",
+            # "Navigate to the object that is <not><desc>",
+            # "Find the object that is <not><desc>",
             "Pick up the object that is <not><desc>",
-            "The goal is object that is <not><desc>",
-            "The object that is <not><desc> is the goal",
+            # "The goal is object that is <not><desc>",
+            # "The object that is <not><desc> is the goal",
```

Comment:
This is a fairly impactful change, just confirming that we want to focus on this single task template in the near future. I would also put in the docstring that this env is focused on pickup only (for now).